CMakeLists.txt

```diff
@@ -1,8 +1,30 @@
-cmake_minimum_required(VERSION 3.13)
-PROJECT (dejavu_cpp_port)
+#./CMakeLists.txt
+cmake_minimum_required(VERSION 3.2 FATAL_ERROR)
+cmake_policy(VERSION 3.2)
+
+project(dejavu_cpp_port
+    VERSION "1.0.0"
```

Comment:
There is no published version: https://github.com/salsowelim/dejavu_cpp_port/tags

Reply:
fair enough